vendors/nordic/boards/nrf52840-dk/ports/common_io/config/iot_flash_config_defaults.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * FreeRTOS
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * http://aws.amazon.com/freertos
+ * http://www.FreeRTOS.org
+ */
+
+/**
+ * @file   iot_flash_config_defaults.h
+ * @brief  Default settings for FLASH
+ */
+
+#ifndef _AWS_COMMON_IO_FLASH_CONFIG_DEFAULTS_H_
+#define _AWS_COMMON_IO_FLASH_CONFIG_DEFAULTS_H_
+
+#ifndef IOT_LOG_LEVEL_IOT_FLASH
+    #define IOT_LOG_LEVEL_IOT_FLASH    IOT_LOG_NONE
+#endif
+
+/* User application base address. Note, iot_flash.h API does not check against this */
+#ifndef IOT_COMMON_IO_FLASH_BASE_ADDRESS
+    #define IOT_COMMON_IO_FLASH_BASE_ADDRESS    0x00000000
```

Comment:
Is this a vendor parameter?  Should this be in some sort of vendor flash defaults to separate vendor defined constants from FreeRTOS constants and User constants?